.gitignore

```diff
@@ -14,3 +14,4 @@ yarn-error.log
 .hyper.js
 .hyper_plugins
 
+.vscode/settings.json
```

Comment:
Is this needed?

Reply:
"needed", contributors will have this file if they work with VSCode so ignoring it makes sure it is not commited by accident

Reply:
This should be in your own global gitignore. Only project related ignores should be here.

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c